nuget/lib/dependabot/nuget/file_updater/project_file_declaration_finder.rb

```diff
@@ -95,6 +111,71 @@ def declaring_file
 
           raise "No file found with name #{filename}!"
         end
+
+        def sdk_import_strings
+          sdk_strings(SDK_IMPORT_REGEX, "Import", "Sdk", "Version")
+        end
+
+        def parse_element(string, name)
+          xml = string
+          xml += "</#{name}>" unless string.end_with?("/>")
```

Comment:
I do not know. It has been a long time since I created the PR, but I _think_ I copied what was already there for other processing (like `PackageReference`).